src/store/singing.ts

```diff
@@ -2079,12 +2064,22 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
             }
           }
 
+          const shouldPlays = shouldPlayTracks(state.tracks);
+
           for (const [i, trackId] of state.trackOrder.entries()) {
             const track = getOrThrow(state.tracks, trackId);
             if (!track.singer) {
               continue;
             }
 
+            // ミュート/ソロをエクスポートするかどうかとして反映する
```

Comment:
ここのコメント難しいですね。。。ちょっと考えてみたのでもし気に入ったら。。
```suggestion
            // ミュート/ソロにより再生されないトラックは除外
```